sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -1,21 +1,20 @@
 # Release History
 
-## 7.7.0-beta.1 (Unreleased)
-
+## 7.7.0 (2022-03-17)
 ### Features Added
 - Added support for sending/receiving messages with `Duration`, `OffsetDateTime` and `URI` in `applicationProperties`.
 - Updated the receiver to retry to obtain a new connection if the RequestResponseChannel in the current connection is disposed.
-- Added getter methods to `ServiceBusProcessorClient` to get the queue, topic and subscription names associated with 
+- Added getter methods to `ServiceBusProcessorClient` to get the queue, topic and subscription names associated with
   the processor.
-  
-### Breaking Changes
 
 ### Bugs Fixed
 - Removed the incorrect use of lock primitives from `ServiceBusMessageBatch.tryAddMessage()` implementation and documented that this API is not thread-safe. ([#25910](https://github.com/Azure/azure-sdk-for-java/issues/25910))
 - Fixed incorrect ordering of message when sent as a batch. ([#25112](https://github.com/Azure/azure-sdk-for-java/issues/25112), [#25599](https://github.com/Azure/azure-sdk-for-java/issues/25599))
 - Fixed a bug that messages come from azure-sdk-for-net with `DescribedType` cannot be deserialized. ([#26065](https://github.com/Azure/azure-sdk-for-java/issues/26065))
 
-### Other Changes
```

Comment:
nit: Need to add this back above `#### Dependency Updates`

Reply:
The norm seems to be removing any empty headers in the changelog, so I was just sticking to that and removing it

Reply:
"Dependency Updates" is a sub-section of "Other Changes". So, this section isn't empty.

Reply:
Generally, `#### Dependency Updates` has gone under `### Other Changes`, has the guideline on that changed?

Reply:
Apparently not in servicebus 😁 It has always stayed as this sub section header, but never under an actual header that made sense. Is there a document / wiki on this? Would love to go through and standardize the change log